stubs/app/client/ui/ProductUpgrades.ts

```diff
@@ -1 +1,4 @@
-export * from 'app/client/ui/ProductUpgradesStub';
+// export * from 'app/client/ui/ProductUpgradesStub';
```

Comment:
Dead code :)

Reply:
What I mean is that we may just remove the line entirely instead of leaving the comment (unless there is some reason to do that?).